### PR TITLE
Implementation and test of api namespaces for time entries #13

### PR DESCRIPTION
**Pull request by EliuX**
This PR:
- Creates the implementation for the API namespace for time entries
- Creates the tests for the API namespace for time entries
- Replaces the status codes for constants
- Improves documentation of expected status codes for the API namespace for projects and time entries

### CODE REVIEW

File: time_tracker_api/api.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 from faker import Faker
 from flask import current_app as app
 from flask_restplus import Api, fields
+from flask_restplus._http import HTTPStatus
+from flask_restplus.fields import Raw, to_marshallable_type, MarshallingError
+from werkzeug.exceptions import Conflict
```
Comment (Angeluz-07) [inline time_tracker_api/api.py]: I don't see how the two last imports are being used.

Comment (EliuX) [reply time_tracker_api/api.py]: Sure,  I will remove them

File: time_tracker_api/sql_repository.py [hunk 2]
```diff
@@ -27,20 +29,15 @@ def init_app(app: Flask) -> None:
     db = SQLAlchemy(app)
 
     global SQLModel
-
-    class SQLModelClass(DatabaseModel):
-        def to_dto(self):
-            return self
-
-    SQLModel = SQLModelClass
+    SQLModel = DatabaseModel
```
Comment (Angeluz-07) [inline time_tracker_api/sql_repository.py]: Checking at changes of `DatabaseModel` Class, and tracing use of `SQLModel` It seems to me that these two classes are not useful anymore.

File: time_tracker_api/database.py [hunk 3]
```diff
@@ -9,60 +9,47 @@
...
     def fresh(self):
-        raise NotImplementedError
+        raise NotImplementedError  # pragma: no cover
 
     def __call__(self, *args, **kwargs):
-        self.run()
+        self.run()  # pragma: no cover
 
 
 class DatabaseModel:
```
Comment (Angeluz-07) [inline time_tracker_api/database.py]: As I said in another comment. https://github.com/ioet/time-tracker-api/pull/43/files#r398215021 I think this class is not needed anymore.

File: time_tracker_api/sql_repository.py (hunk 2 above)
Comment (EliuX) [reply time_tracker_api/sql_repository.py]: Whatever particularity that has to be inherited by the SQLModels should be done throw the SQL model. The `DatabaseModel` is platform-agnostic. Before we had this `to_dto` function that was required by such models. This is basically an encapsulation.

File: time_tracker_api/database.py (hunk 3 above)
Comment (EliuX) [reply time_tracker_api/database.py]: Ok, I will remove then ;)

Comment (EliuX) [reply time_tracker_api/database.py]: Done

**Pull request merged**